py/README

```diff
@@ -14,20 +12,20 @@ The `selenium` package is used automate web browser interaction from Python.
 +-----------+-----------------------------------------------------------------------------------+
 | **Docs**: | `selenium package API <http://selenium.googlecode.com/git/docs/api/py/api.html>`_ |
 +-----------+-----------------------------------------------------------------------------------+
-| **Dev**:  | https://github.com/SeleniumHQ/selenium                                               |
+| **Dev**:  | https://github.com/SeleniumHQ/selenium                                            |
```

Comment:
I would recommend switching the whole table markup to the [simple table markup style](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#simple-tables). The grid tables are in general very fragile (read: error prone).